app/helpers/timer_sessions_helper.rb

```diff
@@ -57,9 +57,12 @@ def sum_work_hours(timer_sessions)
   def issue_information(issue)
     subject = issue.subject
     issue_information = issue.id.to_s
```

Comment:
`issue_information`? what?
```suggestion
issue_id = issue.id.to_s
```

Reply:
no idea, thanks for pointing it out